packages/sources/avalanche-platform/schemas/env.json

```
@@ -0,0 +1,19 @@
+{
+  "$id": "https://external-adapters.chainlinklabs.com/schemas/example-source-adapter.json",
+  "title": "Example Source Adapter",
+  "description": "Example Source Adapter (this title and description were pulled from source/schemas/env.json)",
+  "required": ["P_CHAIN_RPC_URL"],
+  "type": "object",
+  "properties": {
+    "P_CHAIN_RPC_URL": {
+      "type": "string",
+      "description": "Full RPC URL for the avalanche platform chain",
```

Just to avoid any confusion, could you maybe add an example here? (My example below is just an example, I don't know if the path/port is right)

```suggestion
      "description": "Full RPC URL for the avalanche platform chain (e.g. http://localhost:5000/ext/bc/p)",
```